README.md

```diff
@@ -84,6 +84,23 @@ curl -X POST -H 'Content-Type: application/json' -d '{"jsonrpc":"2.0","id":"2","
 The expected response should be `{"result":"0x12a","jsonrpc":"2.0","id":"2"}`
 Where the `result` value matches the .env `CHAIN_ID` configuration value or the current deault value of `298`
 
+### Helm Chart
+
+In this repo there is a `helm-chart` directory that contains the configurations for deploy Hedera's json-rpc relay to a K8s cluster.
+To get started install the helm chart:
+```
+helm install hedera-json-rpc-relay ./helm-chart --debug
+```
+
+To see the values that have been deployed:
+```
+helm show values hedera-json-rpc-relay
+```
+Deploy an installation with custom values file:
```

Comment:
Can you provide an example of a custom file that lets you specify the current values in the `.env` file
This is how people would customize which env and which account the relay uses.

See the `### Configuration` section as an example above

Reply:
I've added secret with mount point to support the .env the same way it is described here.  Also added a section in the README about setting up locally on minikube environment.

Reply:
Lovely